browser/src/control/Control.UserList.js

```diff
@@ -139,8 +140,21 @@ L.Control.UserList = L.Control.extend({
 
 		var that = this;
 
+		var headerUserList = this.options.listUser.slice(-this.options.userLimitHeader);
+
+		// Remove users that should no longer be in the header
+		Array.from(document.querySelectorAll('#userListSummary [data-view-id]')).map(function(element) {
```

Comment:
@eszkadev I wasn't fully sure if I could already use arrow functions within this code parts. Those should be widely adopted if IE11 is not supported anymore: https://caniuse.com/arrow-functions

Maybe you can quickly tell if switching to those would be ok, that would make the code a bit less tangled :)

Reply:
As far as I can remember we have dropped IE11 support since co-2021

Reply:
Decided to stick with the current one as no other .js file uses arrow functions